src/lib.rs

```diff
@@ -79,15 +81,15 @@ pub trait CopyAddress {
     ///
     /// If [`copy_address`] is already defined, then we can provide a standard implementation that
     /// will work across all operating systems.
-    fn get_offset(&self, offsets: &[usize]) -> std::io::Result<usize> {
+    fn get_offset(&self, offsets: &[usize], arch: Architecture) -> std::io::Result<usize> {
         // Look ma! No unsafes!
         let mut offset: usize = 0;
         let noffsets: usize = offsets.len();
+        let mut copy: Vec<u8> = vec![0; arch as usize];
```

Comment:
I'd recommend keeping this vector declaration consistent with the one in `data_member.rs`, so have it be something like `let mut copy = vec![0_u8; arch as usize];`